web/package.json

```diff
@@ -95,47 +95,47 @@
     "@babel/preset-flow": "7.12.1",
     "@babel/preset-react": "7.12.10",
     "@babel/register": "7.12.10",
-    "@testing-library/jest-dom": "^5.11.5",
-    "@testing-library/react": "^11.0.4",
-    "assets-webpack-plugin": "^3.9.10",
+    "@testing-library/jest-dom": "^5.11.9",
+    "@testing-library/react": "^11.2.3",
+    "assets-webpack-plugin": "^7.0.0",
     "babel-jest": "^25.5.1",
     "babel-loader": "^8.0.5",
     "babel-plugin-lodash": "^3.3.4",
     "babel-plugin-styled-components": "^1.10.0",
     "clean-webpack-plugin": "^3.0.0",
-    "copy-webpack-plugin": "^5.1.1",
-    "css-loader": "^3.4.2",
+    "copy-webpack-plugin": "^7.0.0",
+    "css-loader": "^5.0.1",
     "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.2",
+    "enzyme-adapter-react-16": "^1.15.6",
     "enzyme-to-json": "^3.4.3",
-    "es-check": "^5.1.0",
-    "file-loader": "^5.0.2",
+    "es-check": "^5.2.0",
+    "file-loader": "^6.2.0",
     "flow-bin": "~0.117.0",
     "flow-typed": "^2.5.1",
-    "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^3.2.0",
-    "htmlparser2": "~3.9.2",
+    "html-loader": "^1.3.2",
+    "html-webpack-plugin": "^4.5.1",
+    "htmlparser2": "^6.0.0",
     "identity-obj-proxy": "^3.0.0",
     "img-loader": "^3.0.1",
     "jest": "^26.6.3",
     "jest-fetch-mock": "^3.0.1",
     "jest-junit": "^12.0.0",
-    "lolex": "^5.1.2",
+    "lolex": "^6.0.0",
     "moment-locales-webpack-plugin": "^1.2.0",
     "moment-timezone-data-webpack-plugin": "^1.3.0",
     "prop-types": "^15.7.2",
     "raf": "^3.4.1",
     "react-hot-loader": "^4.12.19",
     "redux-mock-store": "^1.5.3",
-    "style-loader": "^1.1.3",
-    "stylelint": "^13.0.0",
+    "style-loader": "^2.0.0",
+    "stylelint": "^13.9.0",
     "stylelint-config-styled-components": "^0.1.1",
     "stylelint-order": "^4.0.0",
     "stylelint-processor-styled-components": "^1.5.2",
-    "url-loader": "^3.0.0",
-    "webpack": "^4.43.0",
-    "webpack-bundle-analyzer": "^3.8.0",
-    "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.11.0"
+    "url-loader": "^4.1.1",
+    "webpack": "^5.16.0",
+    "webpack-bundle-analyzer": "^4.3.0",
+    "webpack-cli": "^4.4.0",
+    "webpack-dev-server": "^3.11.2"
```

Comment:
Is this still needed as you use `webpack serve` now instead?

Reply:
> The build is currently failing because jest is looking in the node_modules directory to run tests.
> 
> Maybe excluding node_modules is enough to fix it.

Added a new commit, but only for the problematic files

Reply:
the "serve" command also uses webpack-dev-server